### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by alb433**

### CODE REVIEW

**Comment by rgcmme** [comment]
* The name of the directory of the HandsOn is not correct.
* All the dataset requirements must be in a single HTML document.

Implement these changes and the pull request will be updated with them.

File: Assignment4/alb433-150213/task06.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1CDaoVZITuPEdiGoGHduyEojTI6tJ5hrG
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/alb433-150213/task06.py:12]: this won't work in Python. Please comment this line

File: Assignment4/alb433-150213/task07.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
+"""**TASK 7.1: List all subclasses of "Person" with RDFLib and SPARQL**"""
+
```
Comment (dgarijo) [inline Assignment4/alb433-150213/task07.py:25]: Missing how to do this in rdflib

File: Assignment4/alb433-150213/task07.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+''', 
+  initNs = {"rdfs": RDFS,"rdf": RDF, "ns": ns}
+)    
+for s in g.query(q1):
+  print(s)
+
+"""**TASK 7.3: List all individuals of "Person" and all their properties including their class with RDFLib and SPARQL**
+
+"""
+
```
Comment (dgarijo) [inline Assignment4/alb433-150213/task07.py:71]: Missing how to do 7.3 in SPARQL

**Comment by dgarijo** [comment]
See missing/errors above for assignment 4

**Review by dgarijo — CHANGES_REQUESTED** [review]
See inline comments regarding the pending issues

**Comment by dgarijo** [comment]
Requested changes have not been implemented. Closing this PR

**Pull request closed without merging**